docs/plugins/gatsby-remark-relative-linker/functions.js

```diff
@@ -0,0 +1,59 @@
+const sibling = "./";
+const parent = "../";
```

Comment:
shall we make all const upper cases (or all lower cases -- to follow whatever the convention is in JS)